src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -138,6 +136,8 @@ export default class SpriteRenderer extends ObjectRenderer
 
         this.vao = this.vaos[0];
         this.currentBlendMode = 99999;
+
+        this.boundTextures = new Array(this.MAX_TEXTURES);
```

Comment:
Yeah the property just needs to exist at construction time. Without doing `this.boundTextures = null` or something, it will perform a Hidden Class transition here, and we want to avoid that.